platform/android/MapboxGLAndroidSDK/src/main/java/org/maplibre/android/style/layers/PropertyFactory.java

```diff
@@ -1806,7 +1806,7 @@ public static PropertyValue<Expression> symbolSpacing(Expression value) {
   }
 
   /**
-   * If true, the symbols will not cross tile edges to avoid mutual collisions. Recommended in layers that don't have enough padding in the vector tile to prevent collisions, or if it is a point symbol layer placed after a line symbol layer. When using a client that supports global collision detection, like MapLibre GL JS version 0.42.0 or greater, enabling this property is not needed to prevent clipped labels at tile boundaries.
+   * If true, the symbols will not cross tile edges to avoid mutual collisions. Recommended in layers that don't have enough padding in the vector tile to prevent collisions, or if it is a point symbol layer placed after a line symbol layer. When using a client that supports global collision detection, like Mapbox GL JS version 0.42.0 or greater, enabling this property is not needed to prevent clipped labels at tile boundaries.
```

Comment:
Is this correct? going from maplibre to mapbox?

Reply:
Technically there never was a MapLibre GL JS 0.42

Reply:
But yes this is 'correct' it uses an older `v8.json`. We can update the `v8.json` but that should be done in a separate PR and may break other things.